pkg/sink/kafka/options.go

```diff
@@ -588,6 +594,20 @@ func AdjustOptions(
 		}
 	}
 
+	// adjust keepConnAliveInterval by `connections.max.idle.ms` broker config.
+	idleMs, err := admin.GetBrokerConfig(ctx, BrokerConnectionsMaxIdleMsConfigName)
+	if err != nil {
```

Comment:
If cannot fetch the configuration, simply use the default configuration, print a log to indicate the error.